pkg/jaeger/store/trace_query.go

```diff
@@ -130,12 +130,10 @@ func (b *Builder) buildCompleteTraceQuery(traceIDClause string, timeLowRef strin
 	linkTimeClause := ""
 	if timeLowRef != "" {
 		spanTimeClause += " AND s.start_time > " + timeLowRef
-		eventTimeClause += " AND e.time > " + timeLowRef
```

Comment:
why do these need to be removed. This will really kill some optimizations

Reply:
`TestExternalGRPCStorage/FindTraces/Trace_spans_over_multiple_indices` would fail without this fix.

Here is a trace fixture https://github.com/jaegertracing/jaeger/blob/main/plugin/storage/integration/fixtures/traces/multi_index_trace.json

The reason is that event stamp is not with in span time boundary(including upper & lower).

```sql
$ select * from _ps_trace.event;
time                         |trace_id                            |span_id|event_nbr|dropped_tags_count|name|tags|
-----------------------------+------------------------------------+-------+---------+------------------+----+----+
2022-09-26 22:16:31.639 +0530|00000000-0000-0000-0000-000000000005|      1|        0|                 0|    |{}  |
2022-09-26 22:16:31.639 +0530|00000000-0000-0000-0000-000000000005|      1|        1|                 0|    |{}  |
2022-09-26 22:16:31.639 +0530|00000000-0000-0000-0000-000000000005|      2|        0|                 0|    |{}  |
2022-09-26 22:16:31.639 +0530|00000000-0000-0000-0000-000000000005|      2|        1|                 0|    |{}  |


$ select * from _ps_trace.span;
trace_id                            |span_id|parent_span_id|operation_id|start_time                   |end_time                     |duration_ms|instrumentation_lib_id|resource_schema_url_id|event_time|dropped_tags_count|dropped_events_count|dropped_link_count|resource_dropped_tags_count|status_code|trace_state|span_tags|status_message|resource_tags|
------------------------------------+-------+--------------+------------+-----------------------------+-----------------------------+-----------+----------------------+----------------------+----------+------------------+--------------------+------------------+---------------------------+-----------+-----------+---------+--------------+-------------+
00000000-0000-0000-0000-000000000005|      1|              |           1|2022-09-26 05:33:31.639 +0530|2022-09-26 05:33:31.639 +0530|      0.005|                      |                      |empty     |                 0|                   0|                 0|                          0|unset      |           |{}       |              |{"1": 1}     |
00000000-0000-0000-0000-000000000005|      2|              |           2|2022-09-26 05:26:31.639 +0530|2022-09-26 05:26:31.639 +0530|      0.005|                      |                      |empty     |                 0|                   0|                 0|                          0|unset      |           |{}       |              |{"1": 1}     |
```

Reply:
Can't we adjust this with the tracing.max-trace-duration CLI parameter instead?

Reply:
Thanks @cevian , Increasing `tracing.max-trace-duration` seem to work. I will remove this change.